TLM/TLM/Custom/PathFinding/CustomPathFind.cs

```diff
@@ -3052,6 +3120,12 @@ private struct BufferItem {
                             }
                         }
 
+                        if ((nextLaneInfo.vehicleCategory & VehicleInfo.VehicleCategory.PublicTransportRoad) != 0 &&
+                            (nextLaneInfo.vehicleCategory & ~(VehicleInfo.VehicleCategory.Bus | VehicleInfo.VehicleCategory.Trolleybus | VehicleInfo.VehicleCategory.Taxi)) == 0)
```

Comment:
would it be beneficial to call `LaneInfo.vehicleCategory` property only once and cache the return value?

Reply:
I'm not sure, I didn't want to create more noise than necessary or even bugs by accident, especially in Pathfinding so it's just copy-paste.